app/views/catalog/_media_html5_player_default.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if resource.is_a?(MediaResource) && decorated_resource.audio_file_sets.present?%>
+  <% audio_file = decorated_resource.audio_file_sets.first %>
```

Comment:
So this player will be able to play just one of the file sets on the resource?

Reply:
Yeah. I can add a few players, one on top of the other, if it's preferable?

Reply:
I wonder if we should do them as part of the member list if we're going to show all of them. Or we might just want to put them on the fileset page?